src/site/content/en/blog/payment-and-address-form-best-practices/index.md

```diff
@@ -0,0 +1,782 @@
+
+Instead, you should set `type="text"` (or leave off the attribute, since `text` is the default) and 
+specify `inputmode="numeric"`, to get a numeric keyboard on mobile. 
+
+To ensure that users get the right keyboard, some sites still use `type="tel"` for payment card 
+numbers. However, `inputmode` is [very widely supported now](https://caniuse.com/input-inputmode), 
+so you shouldn't have to do that, but check your users' browsers.
+{% endAside %}
+
+{: #avoid-custom-elements}
+
+For dates, try to avoid using custom select elements, since they break the autofill experience if not 
+properly implemented, and don't work on older browsers. For numbers such as birth year, consider 
+using an input element rather than a select, since entering digits manually can be easier and less 
+error prone than selecting from a long drop-down list—especially on mobile. Use `inputmode="numeric"` 
+to ensure the right keyboard on mobile and add validation and format hints with text or a 
+placeholder to make sure the user enters data in the appropriate format.
+
+{% Aside %}
+The [datalist](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist) element enables 
+a user to select from a list of available options and provides matching suggestions as the user enters text. 
+Try out datalist for `text`, `range` and `color` inputs at [simpl.info/datalist](https://simpl.info/datalist).
+For birth year input, you can compare a select with an input and datalist at [datalist-select.glitch.me](https://datalist-select.glitch.me).
+{% endAside %}
+
+#### Use autocomplete to improve accessibility and help users avoid re-entering data {: #autocomplete-attribute}
+
+Using appropriate `autocomplete` values enables browsers to help users by securely storing data and 
+autofilling `input`, `select` and `textarea` values. This is particularly important on mobile, and 
+crucial for avoiding [high form abandonment rates](https://www.zuko.io/blog/8-surprising-insights-from-zukos-benchmarking-data). Autocomplete also provides [multiple accessibility benefits](https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html).
+
+If an appropriate autocomplete value is available for a form field, you should use it. [MDN web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete) has a full list of values and explanations of how to use 
+them correctly.
+
+{: #stable-name-id}
+
+{% Aside %}
+As well as using appropriate autocomplete values, help browsers autofill forms by giving form fields 
+`name` and `id` attributes [stable values](#stable-name-id) that don't change between page loads or 
+website deployments.
+{% endAside %}
+
+{: #billing-address}
+
+By default, set the billing address to be the same as the delivery address. Reduce visual clutter by 
+providing a link to edit the billing address (or use [summary and details elements](https://simpl.info/details/)) 
+rather than displaying the billing address in a form.
```

Comment:
```suggestion
By default, set the billing address to be the same as the delivery address. Reduce visual clutter by 
providing a link to edit the billing address (or use [`summary` and `details` elements](https://simpl.info/details/)) 
rather than displaying the billing address in a form.
```